### DISCUSSION TITLE
Can you pls add Müller and GamesOnly.at?

### DISCUSSION THREAD

**Discussion by R3PTIL3** [jef/streetmerchant#1070] [Ideas and feature requests]
Thx

**Comment by maglat** [comment]
would be great!

**Comment by ithahn** [comment]
Would be great if they getting added. But probably the easiest way to add them manually by ourselves...

**Comment by ithahn** [comment]
Euronics would be good as well.

			brand: 'sony',
			model: 'ps5 console',
			series: 'sonyps5c',
			url:
				'https://www.euronics.de/spiele-und-konsolen-film-und-musik/spiele-und-konsolen/playstation-5/spielekonsole/playstation-5-konsole-4061856837826'
		}

**Comment by ithahn** [comment]
Additional possible Stores/Links:
Müller
DE:
https://mueller.vorbestellservice.de/playstation-5/195/playstation-5?number=MVBS10199.1&c=14
Disk:
https://mueller.vorbestellservice.de/playstation-5/195/playstation-5?number=MVBS10199&c=14

Medimax
Disk:
https://www.medimax.de/p/1315336/play-station-5-825gb-ssd
DE:
https://www.medimax.de/p/1315337/play-station-5-digital-edition-825gb-ssd
